htdocs/commande/note.php

```diff
@@ -137,7 +137,13 @@
 
 
 	$cssclass = "titlefield";
-	include DOL_DOCUMENT_ROOT.'/core/tpl/notes.tpl.php';
+	$dirtpls = array_merge($conf->modules_parts['tpl'], array('/core/tpl'));
```

Comment:
If $conf->modules_parts['tpl'] is empty,what if value for $dirtpls ?
May be this can avoidtrouble ?
$dirtpls = array('/core/tpl');
if ($conf->modules_parts['tpl']) {
     $dirtpls = array_merge($conf->modules_parts['tpl'], $dirtpls);
}

Reply:
Yes, this makes sense. Initializing $dirtpls with /core/tpl as a default ensures there's always a valid path, even if $conf->modules_parts['tpl'] is empty.
I make a update, thanks